fluffydiscord/roadrunner-symfony-bundle/2.3/config/packages/fluffy_discord_road_runner.yaml

```diff
@@ -0,0 +1,56 @@
+framework:
+    session:
+        # Fix for Symfony not handling sessions properly in worker mode
+        storage_factory_id: FluffyDiscord\RoadRunnerBundle\Session\WorkerSessionStorageFactory
```

Comment:
I don't think this can be part of the recipe.

Reply:
removed